include/common_package/commands.hpp

```diff
@@ -0,0 +1,421 @@
+#pragma once
+
+#include <cinttypes>
+#include <map>
+#include <nlohmann/json.hpp>
+#include <optional>
+#include <stdexcept>
+#include <string>
+#include <unordered_set>
```

Comment:
It would be cool if this could be sorted by std and custom and 3rd party includes in order to achieve increased legibility.

Reply:
This is something the autoformatter does and I have 0 control over it.
It applies an alphabetic order.

Reply:
You could discuss this with @matixloc. I will resolve this comment as it this is subject to further deeper overall discussion.

Reply:
You can use the following for clang-format in the imports:
// clang-format off
...
// clang-format on
And because of the formatting style, petros has defined it this way. I think because it corresponds to the ros standard formatting style in most cases. If you want to change it, it's best to talk to him.

Reply:
I don't want to change anything, because it is already implemented and I don't think it is worth my time.

Reply:
If you split the imports into groups, that are separated by a space it should keep them separate, that's at least how I do it.

Reply:
Ok, will keep it in mind and do it next time